cluster/gce/win1803/k8s-node-setup.psm1

```diff
@@ -995,6 +1006,23 @@ function Start-WorkerServices {
   # Figure out how to change the directory that the kubelet monitors for new
   # pod manifests.
 
+
+
+  # We configure the service to restart on failure, after 10s wait. We reset the restart
+  # count to 0 each time, so we re-use our restart/10000 action on each failure.
+  # Note it currently restarts even when explicitly stopped, you have to delete the service
+  # entry to *really* kill it (e.g. `sc.exe delete kubelet`). See issue #72900.
+  if (Get-Process | Where-Object Name -eq "kubelet") {
```

Comment:
Do we want to check the process name, or just check the service, e.g., `get-service`? I think the latter probably sufficient.

Reply:
This covers both services and arbitrary processes, which might be better protection against the unexpected?

Reply:
Ack. I was thinking that we could check whether the service exists, and delete them or skip re-creating if needed.

Checking the process is good for disastrous cases I guess.